packages/react-component-library/src/components/ClassificationBar/ClassificationBar.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { StoryFn, Meta } from '@storybook/react'
+
+import { ClassificationBar } from './ClassificationBar'
+
+export default {
+  title: 'ClassificationBar',
```

Comment:
We need to prefix this so it sits in the right place in the updated Storybook:

```suggestion
  title: 'Components/ClassificationBar',
```